util/zenfs.cc

```diff
@@ -481,6 +481,28 @@ int zenfs_tool_dump() {
   return 0;
 }
 
+int zenfs_tool_fsinfo() {
+  Status s;
+  std::unique_ptr<ZonedBlockDevice> zbd = zbd_open(true, false);
+  if (!zbd) return 1;
+  ZonedBlockDevice *zbdRaw = zbd.get();
```

Comment:
It is sort of an anti pattern to take the raw value of a `unique_ptr` before passing a reference to a function. I see that this is the general pattern used, but we should probably change `zbd_open` to just return a raw pointer. We are not really using `unique_ptr` at all, we are just working around it.

Reply:
Agree with you, however this is a non issue for this PR now because `zbdRaw` was not in use, so I removed it.